### PR TITLE
Write a README for autoconfigbrancher

### PR DESCRIPTION
**Pull request by petr-muller**
- Write a README for autoconfigbrancher
- autoconfigbrancher: simplify a conditional

### CODE REVIEW

File: cmd/autoconfigbrancher/main.go [hunk 1]
```diff
@@ -282,9 +282,7 @@ func runSteps(steps []step, author string, stdout, stderr io.Writer) (needsPushi
 			return false, fmt.Errorf("failed to run command and commit the changes: %w", err)
 		}
 
-		if committed {
-			didCommit = didCommit || true
-		}
+		didCommit = didCommit || committed
```
Comment (hongkailiu) [inline cmd/autoconfigbrancher/main.go:285]: NIT: I think this one is simpler to follow:

```
if committed {
			didCommit = true
		}
```

File: cmd/autoconfigbrancher/README.md [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+It iterates over a (hardcoded) sequence of steps that each calls one of the tools that modify some part of the CI
+config. After each step, if there are changes in the config, the changes are committed. If there was at least one new
+commit, the new series of commits is pushed into a new or existing PR using
+the [`bumper` package from test-infra](https://github.com/kubernetes/test-infra/blob/master/prow/cmd/generic-autobumper/bumper/bumper.go)
+.
+
+## How is it deployed
+
+The periodic
+job [periodic-prow-auto-config-brancher](https://prow.ci.openshift.org/?job=periodic-prow-auto-config-brancher) ([definition](https://github.com/openshift/release/blob/master/ci-operator/jobs/infra-periodics.yaml#L828-L875))
```
Comment (hongkailiu) [inline cmd/autoconfigbrancher/README.md:64]: nit: this one is not a permanent address.

**Review by hongkailiu — APPROVED** [review]

**Comment by hongkailiu** [comment]
/hold

chance if you like to address NITs.
Feel free to cancel otherwise.

**Comment by hongkailiu** [comment]
shrug

My hold is not quick enough

**Comment by petr-muller** [comment]
good catch about the nonpermanent link, will follow up

**Pull request merged**